reporter/src/main/kotlin/HowToFixTextProvider.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.reporter
+
+import org.ossreviewtoolkit.model.OrtIssue
+
+/**
+ * Resolves how-to-fix texts in Markdown format for any given [OrtIssue].
+ */
+interface HowToFixTextProvider {
+    /**
+     * Return a Markdown text describing how to fix the given [issue]. Non-null return values override the default
+     * how-to-fix texts, while a null value keeps the default.
```

Comment:
What default? Do you plan to add default how to fix messages?

Reply:
They are empty by default currently, but that could be changed.
When it comes to defining the actual how-to-fix texts, I believe it was you who proposed to add as much as possible to ORT. If that was done the default would no more be empty.